wallet-connect-test-bench/front-end/src/Root.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { WithWalletConnector } from '@concordium/react-components';
+import Main from './Main';
+import { TESTNET } from './constants';
+
+/**
+ * Connect to wallet, setup application state context, and render children when the wallet API is ready for use.
+ */
+export default function Root() {
+    return (
+        <div>
+            <main>
+                <WithWalletConnector network={TESTNET}>{(props) => <Main {...props} />}</WithWalletConnector>
```

Comment:
Is it possible to improve the UI a bit so that 

1. It is limited in width and does not just extend to window width.
2. Add some clearer separator between the output on the right and the actions available on the left. The buttons should be narrower as well so that they are easier to get an overview of.
3. Can you visually separate (e.g., in boxes) the different action types (checking parameters vs. return values)
4. Can you add a paragraph or two on the page explaining what the user/tester should expect
5. If not too inconvenient, can you display values in CCD (with a decimal point)? It's quite challenging to read them with so many decimals :)

Reply:
I made several improvements to the UI.